stages/org.osbuild.tar

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
+"""
+Assembles the tree into a tar archive named `filename`.
+
+Uses the buildhost's `tar` command, like: `tar -cf $FILENAME -C $TREE`
+
+The compression of the tar archive if determined by the suffix, i.e.
+the `--auto-compress` option is used. See tar(1) for details.
```

Comment:
Originally we avoided this to avoid the pass-through to `tar(1)`. The thinking being that if we pass through options without having an allow-list we could never change the underlying implementation. Might be a weak argument though, totally up to you, just thought I'd give the context.

Reply:
Fair, I was thrown on by the "select to compression via an options", and then there were so many options, and "you have to be careful the file name matches what you chosen at the compression option". We could introduce a pattern matcher to only allow suffixes we like though.

Reply:
Maybe, we should just have a compression stage? But we could also have that additionally, one once to get fancy on compression options.

Reply:
Ah, tar complains if the suffix doesn't match? I'd be fine with leaving it as is to be honest.